entity-management-web/src/main/java/eu/europeana/entitymanagement/web/service/EntityRecordService.java

```diff
@@ -1116,9 +1121,12 @@ private void setEuropeanaMetadata(
     europeanaProxy.setProxyId(getEuropeanaProxyId(entityId));
     europeanaProxy.setProxyFor(entityId);
     europeanaProxy.setProxyIn(europeanaAggr);
-
     europeanaProxy.setEntity(europeanaProxyMetadata);
 
+    // set datasource sameAs (if present) in Europeana Proxy
+    if (datasourceResponse != null && datasourceResponse.getSameReferenceLinks() != null) {
```

Comment:
it is not the sameAs of the external datasource which needs to be added to the same as.

Reply:
eventually, in case of wikidata redirections, we should add also the new wikidata ID to same as